knife-zero.gemspec

```diff
@@ -1,32 +1,32 @@
-# coding: utf-8
-lib = File.expand_path('../lib', __FILE__)
+lib = File.expand_path('lib', __dir__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'knife-zero/version'
 
 Gem::Specification.new do |spec|
-  spec.name          = "knife-zero"
+  spec.name          = 'knife-zero'
   spec.version       = Knife::Zero::VERSION
-  spec.authors       = ["sawanoboly"]
-  spec.email         = ["sawanoboriyu@higanworks.com"]
-  spec.summary       = %q{Run chef-client at remote node with chef-zero(local-mode) via HTTP over SSH port fowarding.}
-  spec.description   = %q{Run chef-client at remote node with chef-zero(local-mode) via HTTP over SSH port fowarding.}
-  spec.homepage      = "http://knife-zero.github.io"
-  spec.license       = "Apache-2.0"
+  spec.authors       = ['sawanoboly']
+  spec.email         = ['sawanoboriyu@higanworks.com']
+  spec.summary       = 'Run chef-client at remote node with chef-zero(local-mode) via HTTP over SSH port fowarding.'
+  spec.description   = 'Run chef-client at remote node with chef-zero(local-mode) via HTTP over SSH port fowarding.'
+  spec.homepage      = 'http://knife-zero.github.io'
+  spec.license       = 'Apache-2.0'
+  spec.required_ruby_version = '>= 2.5.0'
```

Comment:
__[RuboCop]__ Gemspec/RequiredRubyVersion: `required_ruby_version` (2.5, declared in knife-zero.gemspec) and `TargetRubyVersion` (2.3, which may be specified in .rubocop.yml) should be equal. ([link](https://sider.review/gh/repos/23179315/pulls/125#issue-4173649))

<sub>posted by Sider</sub>